docs/install-and-run.md

```diff
@@ -254,14 +254,27 @@ steps in your setup:
     #import <EarlGrey/EarlGrey.h>
    ```
 
-2. Download [`EarlGrey.swift`](https://github.com/google/EarlGrey/tree/master/gem/lib/earlgrey/files/EarlGrey.swift) and add it to your test bundle. The file contains
+2. In case you're using the [earlgrey gem](https://rubygems.org/gems/earlgrey), then you need to link to
```

Comment:
formatting is off (view the rendered md).

Organization wise, if you could bold the two ways of including earlgrey.swift, it would be clearer.

2. Include EarlGrey.swift in your project.

  **If using earlgrey gem:**
  `<command to include>`
  **If doing manual Installation:**
  `<manually link to the file>`